index.html

```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Document</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style.css"/>
   </head>
 
   <body>
```

Comment:
- [ ] Please ensure that in your README file in the Demo section, you have a link to the deployed project. I currently do not see it. Based on this, the next reviewer will have to confirm that the deployed site works as expected. thank you